pipelines/controller/src/main/resources/templates/view.html

```diff
@@ -59,11 +59,133 @@
       var options = {
         mode: 'code'
       };
-      var viewContainer = document.getElementById("jsoneditorView");
-      var resourceContainer = document.getElementById("jsoneditorResource");
+      var viewContainer = document.getElementById('jsoneditorView');
+      var resourceContainer = document.getElementById('jsoneditorResource');
       // We want the editors variables to be global.
       viewEditor = new JSONEditor(viewContainer, options);
       resourceEditor = new JSONEditor(resourceContainer, options);
+      const initialView = {
```

Comment:
Can this be loaded from a file instead?

Reply:
I think it is possible, e.g., by defining a template parameter which is being set in the Spring model (by reading a file). But if you don't mind, for now to keep things simpler, I just keep this copy. WDYT?

Reply:
Ok, this is fine for now.